armotypes/notificationreporttypes.go

```diff
@@ -0,0 +1,151 @@
+package armotypes
+
+type WeeklyReport struct {
```

Comment:
Please add this type  to NotificationsConfig struct (in portalType.go) as "latestReport" field (pointer,omitempty)

It will also be nice to move NotificationsConfig to this file